test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/CustomSearchParamTests.cs

```diff
@@ -186,6 +184,7 @@ public async Task GivenASearchParam_WhenUpdatingParam_ThenResourcesIndexedWithUp
                 Assert.Equal(randomName, param.Value.ToString());
 
                 // When job complete, search for resources using new parameter
+                await Task.Delay(10000);
```

Comment:
>await Task.Delay(10000); [](http://example.com/codeflow?start=16&length=24)

why we wait here and not after we post search parameter?

Reply:
added the wait after job is posted